easybuild/easyblocks/s/score_p.py

```diff
@@ -56,16 +63,27 @@ def configure_step(self, *args, **kwargs):
         # Of those, only Cray is supported right now.
         tc_fam = self.toolchain.toolchain_family()
         if tc_fam != toolchain.CRAYPE:
-            # --with-nocross-compiler-suite=(gcc|ibm|intel|pgi|studio)
+            # since 2022/12 releases: --with-nocross-compiler-suite=(gcc|ibm|intel|oneapi|nvhpc|pgi|clang|aocc|amdclang)
             comp_opts = {
                 # assume that system toolchain uses a system-provided GCC
                 toolchain.SYSTEM: 'gcc',
                 toolchain.GCC: 'gcc',
                 toolchain.IBMCOMP: 'ibm',
                 toolchain.INTELCOMP: 'intel',
-                toolchain.NVHPC: 'pgi',
+                toolchain.NVHPC: 'nvhpc',
                 toolchain.PGI: 'pgi',
             }
+            nvhpc_since = {
+                'Score-P': '8.0',
+                'Scalasca': '2.6.1',
+                'OTF2': '3.0.2',
+                'CubeW': '4.8',
```

Comment:
@geimer Should `CubeW` here be `CubeWriter`?

Reply:
@SebastianAchilles Any thoughts on this?

Reply:
> @geimer Should `CubeW` here be `CubeWriter`?

Yes, indeed...  Will fix.